bazel/repository_locations.bzl

```diff
@@ -484,15 +484,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "zlib",
         project_desc = "zlib compression library",
         project_url = "https://zlib.net",
-        version = "79baebe50e4d6b73ae1f8b603f0ef41300110aa3",
-        # Use the dev branch of zlib to resolve fuzz bugs and out of bound
-        # errors resulting in crashes in zlib 1.2.11.
-        # TODO(asraa): Remove when zlib > 1.2.11 is released.
-        sha256 = "155a8f8c1a753fb05b16a1b0cc0a0a9f61a78e245f9e0da483d13043b3bcbf2e",
+        version = "21767c654d31d2dccdde4330529775c6c5fd5389",
```

Comment:
is it better to use the tag `v1.2.12` instead of the SHA?

Reply:
yep - release version/tags are definitely preferable

Reply:
Done